### ISSUE TITLE
Dates in a Collection are mutated on get, set, add, remove

### ISSUE THREAD

**Issue by jtrein** [openlawteam/openlaw-core#97]
When making modifications to OpenLaw Collections the original date timestamps (in milliseconds UTC) appear to be modified on get, set, add, remove. In my current timezone (GMT+545) the diversion appears to be `20700000`ms which is 5.75 hours `===` GMT 545. The difference makes sense, but I am already providing UTC from [getTime()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime). I am wondering if the DateType is doing some kind of time conversion?

I found this while fixing a FE bug where the Collection was not updating properly when adding/removing. Once I got it working, I noticed the time changing each time I clicked "Add" or the _X_ icon to remove.

```
// * ADD ... *
{"values":{"0":"1551681900000"},"size":2}
// * ADD ... *
{"values":{"0":"1551661200000"},"size":3}
// * ADD ... *
{"values":{"0":"1551640500000"},"size":4}
```
This could be the wrong repo for this bug, so please let me know.

cc: @craigwblake

**Comment by jtrein** [comment]
Just to add, I can't find a reliable way to work around this on the FE, elegantly.

**Comment by jtrein** [comment]
Update: Can't get it to work on the front-end.

**Comment by jtrein** [comment]
@craigwblake @adridadou just checking in to see if you guys could reproduce this issue on your end. A user is asking about it in https://github.com/openlawteam/openlaw-elements/issues/41. The steps are to add a Collection of `Date` or `DateTime` types and you should see the Collection values being "mutated" as we add or remove items from the Collection.

To no avail, I've resorted to trying clever things inside the Elements library to try and resolve this, thinking it may be a bug with my JS code. I have not found a way to solve it on my end. Any help is appreciated!

**Comment by craigwblake-openlaw** [comment]
I haven't looked into this yet, but sounds like the impact is high enough we should prioritize it.  I'll see if I can take a look today.

**Comment by outkaj** [comment]
cc @jeamick re: discussion above

**Comment by jtrein** [comment]
Here's a Loom video of the bug in action when adding to a collection. https://www.loom.com/share/37a7196f215447b5be46f0138356d7bc

**Comment by craigwblake-openlaw** [comment]
This should be fixed in  PR #175

**Comment by craigwblake-openlaw** [comment]
Fix is merged to master.
